doc/dftb+/manual/dftbp.tex

```diff
@@ -3506,6 +3506,142 @@ \subsubsection{Charge Model 5}
+%
+\begin{equation}
+  \left[g_j\right]^m_l =
+  \sum^{N_\text{grid}}_{n=1}
+  w_n
+  Y^m_l(\vec y_n)
+  U^j_n
+  \Psi^j_n
+\end{equation}
+%
+where $\Psi^j_n$ is the solute's potential at the $n$-th grid point.
+
+
+\begin{ptable}
+  \kw{FreeEnergyShift} & r & & 0.0 Eh & \\
+  \kw{Temperature} & r & & 298.15 K & \\
+  \kw{State} & s & & gsolv & \\
+  \kw{AngularGrid} & i & & 230 & \\
+  \kw{Radii} & m & & \kw{vanDerWaalsRadiiD3} & \\
+  \kw{RadScale} & r & & 1.0 & \\
+  \kw{Solver} & m & & \kw{DomainDecomposition} & \\
+\end{ptable}
+
+\begin{description}
+  \item[\is{Solvent}] Descriptors of the solvent, can be load from a database
+    by providing the solvent name as string in the \is{FromName\cb} method or
+    by specifying the necessary values with the \is{FromConstants\cb} method.
+    \is{FromConstants\cb} requires the dielectric constant as real for \kw{Epsilon},
+    the molecular mass in \kw{MolecularMass} \modif{\modtype{mass}} and the
+    solvent density in \kw{Density} \modif{\modtype{massdensity}}.
+    \kw{MolecularMass} and \kw{Density} only affect the calculation if
+    ``reference'' is chosen as state of the solution.
+
+  \item[\is{FreeEnergyShift}] \modif{\modtype{energy}} Shift for free energy
+    calculations.
+
+  \item[\is{Temperature}] Temperature for free energy calculations. Default is
+    ambient temperature: 298.15 K.
+    Only affects the calculation for if ``reference'' or ``mol1bar'' is chosen
+    as state of the solution.
+
+  \item[\is{State}] \modif{\modtype{energy}}
+    Reference state of the solution for free energy calculations.
+    The calculated state shift is added to the free energy shift.
+    Takes ``gsolv'' (default), ``reference'' or ``mol1bar''.
+    The reference state ``gsolv'' corresponds to 1\;l of ideal gas and 1\;l of
+    liquid  solution,
+    ``reference'' corresponds to 1\;bar of ideal gas and 1\;mol/L of liquid
+    solution at infinite dilution,
+    ``mol1bar'' corresponds to 1\;bar ideal gas and 1\;mol/L of liquid solution.
+
+  \item[\is{AngularGrid}]
+    Size of the angular Lebedev--Laikov integration grid.\cite{lebedev1999}
+    The grid size mainly determines the computational cost of solving the
+    COSMO equations, too small grid sizes can lead to significant errors
+    due to missing rotational invariance.
+    Possible values are
+    6, 14, 26, 38, 50, 74, 86, 110, 146, 170, 194, 230, 266, 302, 350,
+    434, 590, 770, 974, 1202, 1454, 1730, 2030, 2354, 2702, 3074, 3470, 3890,
+    4334, 4802, 5294, 5810.
+
+  \item[\is{Radii}] Atomic radii for each element, either takes
+    \is{VanDerWaalsRadiiD3\cb} for DFT-D3 van-der-Waals radii (can be overwritten)
+    or requires to provide \is{Values\cb} for all species.
+    Both methods accept \modif{\modtype{length}} units.
+
+  \item[\is{RadScale}] Allows to scale the provided radii by the provided value.
+    Values greater one between 1.2 and 1.5 are recommended when using van-der-Waals radii.
```

Comment:
I was actually trying to avoid writing out radius or radii here.

Reply:
I see. Usually we try to avoid abbreviations in general, and especially in the first part of a keyword if possible and reasonable (kinda  [explicit over implicit](https://www.python.org/dev/peps/pep-0020/) :wink: )